libs/wingsdk/src/target-tf-aws/bucket.ts

```diff
@@ -21,7 +26,9 @@ export class Bucket extends cloud.BucketBase {
 
     this.public = props.public ?? false;
 
-    this.bucket = new S3Bucket(this, "Default");
+    this.bucket = new S3Bucket(this, "Default", {
+      bucket: this.sanitizeName(`${this.node.id}-${this.node.addr}`),
```

Comment:
Why is this called `bucket` and not something consistent across resources like `name`?

Reply:
The `bucket` property is from S3Bucket class

It's from CdkTF library